Tests/SonarScanner.MSBuild.Common.Test/AnalysisConfig/AnalysisConfigExtensionsTests.cs

```diff
@@ -77,12 +77,22 @@ public void GetConfigValue_WhenSettingIdIsWhitespace_ThrowsArgumentNullException
         public void GetAnalysisSettings_WhenConfigIsNull_ThrowsArgumentNullException()
         {
```

Comment:
I suggest creating a [TestInitialize] method and create the logger there and use everywhere in the class from a private field.

Reply:
it's used only in 6 out of the 28 tests

I'd agree if the proportion would be higher; however, I won't do it just for 6 tests